test/e2e/destroypod_test.go

```diff
@@ -252,9 +253,10 @@ func TestDestroyPodWithRequests(t *testing.T) {
 	pods, err := clients.KubeClient.Kube.CoreV1().Pods(test.ServingNamespace).List(metav1.ListOptions{
 		LabelSelector: fmt.Sprintf("%s=%s", serving.RevisionLabelKey, objects.Revision.Name),
 	})
-	if err != nil || len(pods.Items) != 1 {
-		t.Fatalf("Number of pods is not 1 or an error: %v", err)
+	if err != nil || len(pods.Items) == 0 {
+		t.Fatalf("No pods or error: %v", err)
```

Comment:
No, that won't do :( it's after fatal, i.e. won't be printed.
I'd put it either in the fatal message or _above_ the `if` statement.

Reply:
If we're at line 57 then either we can't list the Pods or there is 0 Pod.

Reply:
I see.